mobile/lib/util/preferences.dart

```diff
@@ -10,6 +11,17 @@ class Preferences {
   static const emailAddress = "emailAddress";
   static const openPosition = "openPosition";
   static const fullBackup = "fullBackup";
+  static const logLevelTrace = "logLevelTrace";
+
+  Future<bool> setLogLevelTrace(bool trace) async {
```

Comment:
Instead of setting it to a boolean you could set it to a `loglevel` or a `string` so that we can support different log levels, but I guess this is good enough for now :)

Reply:
I thought about it, but it would have been a little bit more complex for something we might never add.

Happy to change it once we see the need to add other log levels as well :)